lib/ansible/modules/source_control/git.py

```diff
@@ -919,6 +950,9 @@ def main():
     key_file  = module.params['key_file']
     ssh_opts  = module.params['ssh_opts']
     umask  = module.params['umask']
+    archive = module.params['archive']
+    archive_fmt = module.params['archive_format']
+    archive_file = module.params['archive_file']
```

Comment:
Do we need all three parameters? 

Why not just create an archive if `archive_file` is set (in fact, that could be the `archive` parameter).